src/TypeSpec.Extension/Emitter.Csharp/src/options.ts

```diff
@@ -30,6 +30,7 @@ export interface AzureCSharpEmitterOptions extends CSharpEmitterOptions {
     "examples-dir"?: string;
     "enable-bicep-serialization"?: boolean;
     "clear-output-folder"?: boolean;
+    "suppress-abstract-base-class"?: string[];
```

Comment:
I am not convinced by adding configuration directly to tspconfig.yaml.
This might introduce a long list of classes, and it will flood tspconfig.yaml, which is not user-friendly.